app/uk/gov/hmrc/agentsubscriptionfrontend/config/frontendAppConfig.scala

```diff
@@ -84,16 +82,24 @@ class FrontendAppConfig @Inject()(val environment: Environment, val configuratio
   override val addressLookupContinueUrl: String = getServicesConfStringOrFail(
     "address-lookup-frontend.new-address-callback.url")
   override val surveyRedirectUrl: String = getConfStringOrFail(s"$env.surveyRedirectUrl")
-  override val ssoRedirectUrl: String = getConfStringOrFail(s"$env.sosRedirectUrl")
+
   override val companyAuthSignInUrl: String = getConfStringOrFail(s"$env.companyAuthSignInUrl")
   override val chainedSessionDetailsTtl: Int = getConfIntOrFail(s"$env.mongodb.chainedsessiondetails.ttl")
   override val cacheableSessionDomain: String = getServicesConfStringOrFail("cachable.session-cache.domain")
   override def agentMappingFrontendStartUrl(continueId: String): String =
     s"${getServicesConfStringOrFail("agent-mapping-frontend.external-url")}${getServicesConfStringOrFail(
       "agent-mapping-frontend.start.path")}?continueId=$continueId"
+
+  val ssoRedirectUrl: String = "/government-gateway-registration-frontend?accountType=agent&origin=unknown"
+
   override val ggRegistrationFrontendExternalUrl: String =
     s"${getConfStringOrFail(s"$env.microservice.services.government-gateway-registration-frontend.externalUrl")}$ssoRedirectUrl"
 
+  private val rootContinuePath: String = "/agent-subscription/return-after-gg-creds-created"
```

Comment:
could rename this returnAfterGGCredCreatedsPath or something like that

Reply:
Consider it done :)